core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -228,12 +228,6 @@ public NativeImageBuildItem build(NativeConfig nativeConfig, NativeImageSourceJa
                     final Path finalSources = outputTargetBuildItem.getOutputDirectory().resolve(sources);
                     IoUtils.copy(generatedSources, finalSources);
                     IoUtils.recursiveDelete(generatedSources);
-                } else {
-                    log.warn(
-                            "objcopy executable not found in PATH. Debug symbols therefore cannot be placed into the dedicated directory.");
-                    log.warn("That also means that resulting native executable is larger as it embeds the debug symbols.");
-                }
```

Comment:
Hmmm, it looks you were a bit more aggressive here, we need that `}` back :)

Reply:
I'll fix this shortly.